src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -475,6 +475,56 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP087",
                 "Array and object literals are not allowed here.");
+
+            public ErrorDiagnostic UnableToFindPathForModule() => new ErrorDiagnostic(
+                TextSpan,
+                "BCP088",
+                "Unable to find file path for module.");
+
+            public ErrorDiagnostic ErrorOccurredLoadingModule(string failureMessage) => new ErrorDiagnostic(
+                TextSpan,
+                "BCP089",
+                $"An error occurred loading the module. Received failure \"{failureMessage}\".");
+
+            public ErrorDiagnostic ModulePathInterpolationUnsupported() => new ErrorDiagnostic(
+                TextSpan,
+                "BCP090",
+                "String interpolation is unsupported for specifying the module path.");

```

Comment:
Can we change to "String interpolation is not supported in module paths."?

Reply:
Yup, that's a much more sensible sentence :)